cyral/resource_cyral_repository.go

```diff
@@ -48,6 +49,13 @@ func resourceRepository() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"labels": {
```

Comment:
```suggestion
			"labels": {
				Description: "",
```

Please add the description for this new argument. Also, make sure to update the docs for this resource with the very same description. The documentation will be created automatically from the source code in [this PR](https://github.com/cyralinc/terraform-provider-cyral/pull/183).